content/influxdb/cloud-iox/reference/sql/_index.md

```diff
@@ -9,4 +9,597 @@ menu:
+| SUM()    | Returns the summed value of a column                       |
+| MEAN()   | Returns the mean value of a column                         |
+| MIN()    | Returns the smallest value of the selected column          |
+| MAX()    | Returns the largest value of the selected column          |
+
+#### Examples
+
+```sql
+
+SELECT COUNT("water_level") 
+FROM "h2o_feet"
+
+SELECT AVG("water_level"), "location"
+FROM "h2o_feet" 
+GROUP BY "location"
+
+SELECT SUM("water_level"), "location"
+FROM "h2o_feet" 
+GROUP BY "location"
+```
+
+### Selector functions
+
+Selector functions are unique to InfluxDB. They behave like aggregate functions in that they take a row of data and compute it down to a single value.  However, selectors are unique in that they return a **time value** in addition to the computed value. In short, selectors retrun an aggreagetd value along with a timestamp. 
+
+| Function         | Description                                                     |
+| :--------------- | :-------------------------------------------------------------- |
+| SELECTOR_FIRST() | Returns the first value of a selected column and timestamp.   |
+| SELECTOR_LAST()  | Returns the last value of a selected column and timestamp.     |
+| SELECTOR_MIN()   | Returns the smallest value of a selected column and timestamp. |
+| SELECTOR_MAX()   | Returns the largest value of a selected column and timestamp.  |
+
+#### Examples
+
+```sql
+SELECT 
+SELECTOR_MAX("pH", time)['value'],
+SELECTOR_MAX("pH", time)['time']
+FROM "h2o_pH"
+
+SELECT 
+SELECTOR_LAST("water_level", time)['value'],
+SELECTOR_LAST("water_level", time)['time']
+FROM "h2o_feet"
+WHERE time >= timestamp '2019-09-10T00:00:00Z' AND time <= timestamp '2019-09-19T00:00:00Z'
+
+```
+
+### Time series functions
+
+| Function     | Description                                                                                     |
+| :----------- | :---------------------------------------------------------------------------------------------- |
+| DATE_BIN()   | Bins the input timestamp into a specified interval.                                             |
+| DATE_TRUNC() | Truncates a timestamp expression based on the date part specified, such as hour, day, or month. |
+| DATE_PART()  | Returns the specified part of a date.                                                           |
+| NOW()        | Returns the current time.                                                                       |
+                 
+#### Examples
+
+```sql
+SELECT DATE_BIN(INTERVAL '1 hour', time, '2019-09-18T00:00:00Z'::timestamp),
+SUM(water_level)
+FROM "h2o_feet"
+GROUP BY time
+
+SELECT DATE_TRUNC('month',time) AS "date",
+SUM(water_level)
+FROM "h2o_feet"
+GROUP BY time
+```
+
+### Approximate functions
+
+| Function      | Description                                    |
+| :------------ | :--------------------------------------------- |
+| APPROX_MEDIAN | Returns the approximate median of input values. |
```

Comment:
This is missing functions.

- approx_distinct
- approx_percentile_cont
- approx_percentile_cont_with_weight

Reply:
adding now